sql/tests: TestRandomSyntaxFunctions failed

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/19654340?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/19654340?buildTab=artifacts#/) on release-25.1 @ [1a50865c48b6bd2f3ccfacc7ab2e49142d8e9b47](https://github.com/cockroachdb/cockroach/commits/1a50865c48b6bd2f3ccfacc7ab2e49142d8e9b47):

Fatal error:

```
panic: test timed out after 1h0m0s
running tests:
	TestRandomSyntaxFunctions (49m57s)

```
Stack: 

```
goroutine 6028244 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
* github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).run(0xc0026a0008, {0x8aab408, 0xc015ae4780}, 0xc00346ef00?, 0x0?, 0xc011ae2c80?)
* 	pkg/sql/conn_executor.go:2306 +0x208
* github.com/cockroachdb/cockroach/pkg/sql.(*Server).ServeConn(0xc00e8b9900?, {0x8aab408?, 0xc015ae4780?}, {0xc0282af320?}, 0x2?, 0x0?)
* 	pkg/sql/conn_executor.go:1002 +0xda
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).processCommands(0xc00c919408, {0x8aab408, 0xc015ae4780}, {0x0, 0x4, {0x1, {0x719ba4e, 0x3}, {0xc01ac0a6f0, 0xf}, ...}, ...}, ...)
* 	pkg/sql/pgwire/conn.go:252 +0x4c8
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveImpl.func4()
* 	pkg/sql/pgwire/server.go:1197 +0xec
* created by github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveImpl in goroutine 4038504
* 	pkg/sql/pgwire/server.go:1194 +0x5fb
* 
* goroutine 3836538 [chan send, 11 minutes]:
* github.com/cockroachdb/cockroach/pkg/sql.(*planner).fingerprintSpanFanout.(*planner).fingerprintSpanFanout.func1.func2({0x8aa9db8, 0xc011dc8f50})
* 	pkg/sql/fingerprint_span.go:178 +0x3b9
* github.com/cockroachdb/cockroach/pkg/sql.(*planner).fingerprintSpanFanout.Group.GoCtx.func3()
* 	pkg/util/ctxgroup/ctxgroup.go:189 +0x8b
* golang.org/x/sync/errgroup.(*Group).Go.func1()
* 	external/org_golang_x_sync/errgroup/errgroup.go:78 +0x56
* created by golang.org/x/sync/errgroup.(*Group).Go in goroutine 3836225
* 	external/org_golang_x_sync/errgroup/errgroup.go:75 +0x96
* 
* goroutine 3718987 [select]:
* github.com/cockroachdb/pebble/vfs.(*diskHealthCheckingFile).startTicker.func1()
* 	external/com_github_cockroachdb_pebble/vfs/disk_health.go:254 +0xbc
* created by github.com/cockroachdb/pebble/vfs.(*diskHealthCheckingFile).startTicker in goroutine 3718799
* 	external/com_github_cockroachdb_pebble/vfs/disk_health.go:249 +0x58
* 
* goroutine 3718914 [chan receive, 11 minutes]:
* github.com/cockroachdb/pebble.(*fileCacheShard).releaseLoop(0xc0269a64d0)
* 	external/com_github_cockroachdb_pebble/file_cache.go:399 +0x97
* created by github.com/cockroachdb/pebble.(*fileCacheShard).init in goroutine 3718799
* 	external/com_github_cockroachdb_pebble/file_cache.go:390 +0xe5
* 
* goroutine 6021414 [sync.Cond.Wait, 2 minutes]:
* sync.runtime_notifyListWait(0xc00811a0a8, 0xf4b)
* 	GOROOT/src/runtime/sema.go:569 +0x159
* sync.(*Cond).Wait(0xc00811a000?)
* 	GOROOT/src/sync/cond.go:70 +0x85
* github.com/cockroachdb/pebble/record.(*flusherCond).Wait(...)
* 	external/com_github_cockroachdb_pebble/record/log_writer.go:371
* github.com/cockroachdb/pebble/record.(*LogWriter).flushLoop(0xc00811a000, {0x66724c0?, 0xd3e2460?})
* 	external/com_github_cockroachdb_pebble/record/log_writer.go:645 +0x6ea
* runtime/pprof.Do({0x8aa9bd8?, 0xd3e2460?}, {{0xc001607ce0?, 0xc005ff3f30?, 0xc005ff3fd0?}}, 0xc005ff3fc0)
* 	GOROOT/src/runtime/pprof/runtime.go:51 +0x9d
* github.com/cockroachdb/pebble/record.NewLogWriter.func2()
* 	external/com_github_cockroachdb_pebble/record/log_writer.go:564 +0x56
* created by github.com/cockroachdb/pebble/record.NewLogWriter in goroutine 3719076
* 	external/com_github_cockroachdb_pebble/record/log_writer.go:563 +0x4b8
* 
*
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-50509

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/19661556?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/19661556?buildTab=artifacts#/) on release-25.1 @ [1797174e32d9cd3419c332bf3f8fb9c59e2bf2f5](https://github.com/cockroachdb/cockroach/commits/1797174e32d9cd3419c332bf3f8fb9c59e2bf2f5):

Fatal error:

```
panic: test timed out after 1h0m0s
running tests:
	TestRandomSyntaxFunctions (49m57s)

```
Stack: 

```
goroutine 5843883 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
* sync.runtime_Semacquire(0xcb594a0?)
* 	GOROOT/src/runtime/sema.go:62 +0x25
* sync.(*WaitGroup).Wait(0x8a4d0a0?)
* 	GOROOT/src/sync/waitgroup.go:116 +0x48
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveImpl(0xc00807b930, {0x8aaae78, 0xc01c271ef0}, 0xc0176d2408, 0xc00ac13068, {0x0, 0x4, {0x1, {0x719c76e, 0x3}, ...}, ...}, ...)
* 	pkg/sql/pgwire/server.go:1452 +0xa7d
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn(0xc00807b930, {0x8aaae78, 0xc01c271ef0}, {0x8afb4a0, 0xc01750e388}, {0x0, 0x4, 0x0, 0xc00ac13020, {{{{...}}, ...}, ...}})
* 	pkg/sql/pgwire/server.go:958 +0xb36
* github.com/cockroachdb/cockroach/pkg/server.(*systemServerWrapper).serveConn(0xc00519e358, {0x8aac4c8, 0xc01e282460}, {0x8afb4a0, 0xc01750e388}, {0x0, 0x4, 0x0, 0xc00ac13020, {{{{...}}, ...}, ...}})
* 	pkg/server/server_controller_sql.go:170 +0xf7
* github.com/cockroachdb/cockroach/pkg/server.(*serverController).sqlMux(0xc0220da9a0, {0x8aac4c8, 0xc01e282460}, {0x8afb4a0, 0xc01750e388}, {0x0, 0x4, 0x0, 0xc00ac13020, {{{{...}}, ...}, ...}})
* 	pkg/server/server_controller_sql.go:90 +0x2f8
* github.com/cockroachdb/cockroach/pkg/server.startServeSQL.func1.1({0x8aac4c8, 0xc018bb4a00}, {0x8afbc10, 0xc01ff65db0})
* 	pkg/server/server_sql.go:1990 +0x30c
* github.com/cockroachdb/cockroach/pkg/util/netutil.(*TCPServer).ServeWith.func1({0x8aac4c8, 0xc018bb4a00})
* 	pkg/util/netutil/net.go:186 +0x102
* github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2({0x71c51ec?, 0x0?})
* 	pkg/util/stop/stopper.go:498 +0x1f0
* created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 3586751
* 	pkg/util/stop/stopper.go:488 +0x47c
* 
* goroutine 4795990 [semacquire, 2 minutes]:
* sync.runtime_Semacquire(0xcb594a0?)
* 	GOROOT/src/runtime/sema.go:62 +0x25
* sync.(*WaitGroup).Wait(0x8a4d0a0?)
* 	GOROOT/src/sync/waitgroup.go:116 +0x48
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveImpl(0xc00807b930, {0x8aaae78, 0xc0052154a0}, 0xc018fb4008, 0xc0286ff968, {0x0, 0x4, {0x1, {0x719c76e, 0x3}, ...}, ...}, ...)
* 	pkg/sql/pgwire/server.go:1452 +0xa7d
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn(0xc00807b930, {0x8aaae78, 0xc0052154a0}, {0x8afb4a0, 0xc012342a88}, {0x0, 0x4, 0x0, 0xc0286ff908, {{{{...}}, ...}, ...}})
* 	pkg/sql/pgwire/server.go:958 +0xb36
* github.com/cockroachdb/cockroach/pkg/server.(*systemServerWrapper).serveConn(0xc00519e358, {0x8aac4c8, 0xc01b9825a0}, {0x8afb4a0, 0xc012342a88}, {0x0, 0x4, 0x0, 0xc0286ff908, {{{{...}}, ...}, ...}})
* 	pkg/server/server_controller_sql.go:170 +0xf7
* github.com/cockroachdb/cockroach/pkg/server.(*serverController).sqlMux(0xc0220da9a0, {0x8aac4c8, 0xc01b9825a0}, {0x8afb4a0, 0xc012342a88}, {0x0, 0x4, 0x0, 0xc0286ff908, {{{{...}}, ...}, ...}})
* 	pkg/server/server_controller_sql.go:90 +0x2f8
* github.com/cockroachdb/cockroach/pkg/server.startServeSQL.func1.1({0x8aac4c8, 0xc018bb4a00}, {0x8afbc10, 0xc02bf74370})
* 	pkg/server/server_sql.go:1990 +0x30c
* github.com/cockroachdb/cockroach/pkg/util/netutil.(*TCPServer).ServeWith.func1({0x8aac4c8, 0xc018bb4a00})
* 	pkg/util/netutil/net.go:186 +0x102
* github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2({0x71c51ec?, 0x0?})
* 	pkg/util/stop/stopper.go:498 +0x1f0
* created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 3586751
* 	pkg/util/stop/stopper.go:488 +0x47c
* 
* goroutine 3584875 [chan receive, 10 minutes]:
* github.com/cockroachdb/pebble.(*fileCacheShard).releaseLoop(0xc01f480000)
* 	external/com_github_cockroachdb_pebble/file_cache.go:399 +0x97
* created by github.com/cockroachdb/pebble.(*fileCacheShard).init in goroutine 3584780
* 	external/com_github_cockroachdb_pebble/file_cache.go:390 +0xe5
* 
*
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #146495 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master release-blocker]
- #146423 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-25.2.0-rc release-blocker]
- #146424 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-25.2 release-blocker]
- #146422 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-25.1.6-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/19664338?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/19664338?buildTab=artifacts#/) on release-25.1 @ [1797174e32d9cd3419c332bf3f8fb9c59e2bf2f5](https://github.com/cockroachdb/cockroach/commits/1797174e32d9cd3419c332bf3f8fb9c59e2bf2f5):

Fatal error:

```
panic: test timed out after 1h0m0s
running tests:
	TestRandomSyntaxFunctions (49m57s)

```
Stack: 

```
goroutine 5721131 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
* github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).makeExecPlan(0xc018e98008, {0x8aaae78, 0xc02b3bfa90}, 0xc018e98698)
* 	pkg/sql/conn_executor_exec.go:3188 +0xcb
* github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).dispatchToExecutionEngine(0xc018e98008, {0x8aaae78, 0xc02b3bfa90}, 0xc018e98698, {0x7fb48b6bbdb8, 0xc01b351f60})
* 	pkg/sql/conn_executor_exec.go:2745 +0x670
* github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmtInOpenState(0xc018e98008, {0x8aac4c8?, 0xc01af6e3c0?}, {{0x8acfde0, 0xc02b3bf950}, {0x0, 0x0, 0x0}, {0xc01becd8a8, 0x7d}, ...}, ...)
* 	pkg/sql/conn_executor_exec.go:1038 +0x38e7
* github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmt.func2({0x8aac4c8?, 0xc01af6e3c0?})
* 	pkg/sql/conn_executor_exec.go:164 +0xb0
* github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execWithProfiling(0x8aac4c8?, {0x8aac4c8?, 0xc01af6e3c0?}, {0x8acfde0?, 0xc02b3bf950?}, 0xc01becd8a8?, 0xc01875db08?)
* 	pkg/sql/conn_executor_exec.go:4345 +0x2ec
* github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmt(0xc018e98008, {0x8aac4c8, 0xc01af6e3c0}, {{0x8acfde0, 0xc02b3bf950}, {0x0, 0x0, 0x0}, {0xc01becd8a8, 0x7d}, ...}, ...)
* 	pkg/sql/conn_executor_exec.go:163 +0x885
* github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd.func1({{{0x8acfde0, 0xc02b3bf950}, {0x0, 0x0, 0x0}, {0xc01becd8a8, 0x7d}, 0x0, 0x0}, 0xcc9ae2f42e, ...}, ...)
* 	pkg/sql/conn_executor.go:2384 +0x3dd
* github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd(0xc018e98008)
* 	pkg/sql/conn_executor.go:2389 +0x848
* github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).run(0xc018e98008, {0x8aac4c8, 0xc018735040}, 0x3c?, 0xc0059c1280?, 0xc0286b44e0?)
* 	pkg/sql/conn_executor.go:2306 +0x208
* github.com/cockroachdb/cockroach/pkg/sql.(*Server).ServeConn(0xc014745e00?, {0x8aac4c8?, 0xc018735040?}, {0xc00470b7a0?}, 0x2?, 0x0?)
* 	pkg/sql/conn_executor.go:1002 +0xda
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).processCommands(0xc01b351808, {0x8aac4c8, 0xc018735040}, {0x0, 0x4, {0x1, {0x719c76e, 0x3}, {0xc005b6ae40, 0xf}, ...}, ...}, ...)
* 	pkg/sql/pgwire/conn.go:252 +0x4c8
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveImpl.func4()
* 	pkg/sql/pgwire/server.go:1197 +0xec
* created by github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveImpl in goroutine 5601624
* 	pkg/sql/pgwire/server.go:1194 +0x5fb
* 
* goroutine 5606904 [semacquire, 2 minutes]:
* sync.runtime_Semacquire(0xcb594a0?)
* 	GOROOT/src/runtime/sema.go:62 +0x25
* sync.(*WaitGroup).Wait(0x8a4d0a0?)
* 	GOROOT/src/sync/waitgroup.go:116 +0x48
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveImpl(0xc01c117110, {0x8aaae78, 0xc015638370}, 0xc00636b408, 0xc00d3cc9c0, {0x0, 0x4, {0x1, {0x719c76e, 0x3}, ...}, ...}, ...)
* 	pkg/sql/pgwire/server.go:1452 +0xa7d
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn(0xc01c117110, {0x8aaae78, 0xc015638370}, {0x8afb4a0, 0xc00e52a388}, {0x0, 0x4, 0x0, 0xc00d3cc978, {{{{...}}, ...}, ...}})
* 	pkg/sql/pgwire/server.go:958 +0xb36
* github.com/cockroachdb/cockroach/pkg/server.(*tenantServerWrapper).serveConn(0xc00db97df0, {0x8aac4c8, 0xc0188ac5a0}, {0x8afb4a0, 0xc00e52a388}, {0x0, 0x4, 0x0, 0xc00d3cc978, {{{{...}}, ...}, ...}})
* 	pkg/server/server_controller_sql.go:186 +0xf8
* github.com/cockroachdb/cockroach/pkg/server.(*serverController).sqlMux(0xc00fd0b180, {0x8aac4c8, 0xc0188ac5a0}, {0x8afb4a0, 0xc00e52a388}, {0x0, 0x4, 0x0, 0xc00d3cc978, {{{{...}}, ...}, ...}})
* 	pkg/server/server_controller_sql.go:90 +0x2f8
* github.com/cockroachdb/cockroach/pkg/server.startServeSQL.func1.1({0x8aac4c8, 0xc005a9cc80}, {0x8afbc10, 0xc0137c3940})
* 	pkg/server/server_sql.go:1990 +0x30c
* github.com/cockroachdb/cockroach/pkg/util/netutil.(*TCPServer).ServeWith.func1({0x8aac4c8, 0xc005a9cc80})
* 	pkg/util/netutil/net.go:186 +0x102
* github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2({0x71c51ec?, 0x0?})
* 	pkg/util/stop/stopper.go:498 +0x1f0
* created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 3270522
* 	pkg/util/stop/stopper.go:488 +0x47c
* 
*
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #146495 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-master release-blocker]
- #146424 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-25.2 release-blocker]
- #146423 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-25.2.0-rc release-blocker]
- #146422 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-25.1.6-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/19674443?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/19674443?buildTab=artifacts#/) on release-25.1 @ [7a182f5da5469f091d25b671dc3fba759284fdfc](https://github.com/cockroachdb/cockroach/commits/7a182f5da5469f091d25b671dc3fba759284fdfc):

Fatal error:

```
panic: test timed out after 1h0m0s
running tests:
	TestRandomSyntaxFunctions (49m57s)

```
Stack: 

```
goroutine 5402566 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
* github.com/cockroachdb/pebble/record.(*flusherCond).Wait(...)
* 	external/com_github_cockroachdb_pebble/record/log_writer.go:371
* github.com/cockroachdb/pebble/record.(*LogWriter).flushLoop(0xc0138c2000, {0x6675020?, 0xd3e74e0?})
* 	external/com_github_cockroachdb_pebble/record/log_writer.go:645 +0x6ea
* runtime/pprof.Do({0x8aad1d8?, 0xd3e74e0?}, {{0xc000a16b00?, 0xc00c504fd0?, 0x46041d?}}, 0xc00c504fc0)
* 	GOROOT/src/runtime/pprof/runtime.go:51 +0x9d
* github.com/cockroachdb/pebble/record.NewLogWriter.func2()
* 	external/com_github_cockroachdb_pebble/record/log_writer.go:564 +0x56
* created by github.com/cockroachdb/pebble/record.NewLogWriter in goroutine 2844902
* 	external/com_github_cockroachdb_pebble/record/log_writer.go:563 +0x4b8
* 
* goroutine 2844943 [select]:
* github.com/cockroachdb/cockroach/pkg/util/admission.(*WorkQueue).startClosingEpochs.func1()
* 	pkg/util/admission/work_queue.go:501 +0x14d
* created by github.com/cockroachdb/cockroach/pkg/util/admission.(*WorkQueue).startClosingEpochs in goroutine 2844902
* 	pkg/util/admission/work_queue.go:472 +0x4f
* 
* goroutine 5245276 [chan send, 5 minutes]:
* github.com/cockroachdb/cockroach/pkg/sql.(*planner).fingerprintSpanFanout.(*planner).fingerprintSpanFanout.func1.func2({0x8aad3b8, 0xc042936690})
* 	pkg/sql/fingerprint_span.go:178 +0x3b9
* github.com/cockroachdb/cockroach/pkg/sql.(*planner).fingerprintSpanFanout.Group.GoCtx.func3()
* 	pkg/util/ctxgroup/ctxgroup.go:189 +0x8b
* golang.org/x/sync/errgroup.(*Group).Go.func1()
* 	external/org_golang_x_sync/errgroup/errgroup.go:78 +0x56
* created by golang.org/x/sync/errgroup.(*Group).Go in goroutine 5244698
* 	external/org_golang_x_sync/errgroup/errgroup.go:75 +0x96
* 
* goroutine 4395299 [semacquire, 2 minutes]:
* sync.runtime_Semacquire(0xcb5d4a0?)
* 	GOROOT/src/runtime/sema.go:62 +0x25
* sync.(*WaitGroup).Wait(0x8a4f5c0?)
* 	GOROOT/src/sync/waitgroup.go:116 +0x48
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveImpl(0xc007d504e0, {0x8aad3b8, 0xc028590c80}, 0xc018e92008, 0xc0201d28b8, {0x0, 0x4, {0x1, {0x719e84e, 0x3}, ...}, ...}, ...)
* 	pkg/sql/pgwire/server.go:1452 +0xa7d
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn(0xc007d504e0, {0x8aad3b8, 0xc028590c80}, {0x8afd9e0, 0xc00cd73888}, {0x0, 0x4, 0x0, 0xc0201d2828, {{{{...}}, ...}, ...}})
* 	pkg/sql/pgwire/server.go:958 +0xb36
* github.com/cockroachdb/cockroach/pkg/server.(*systemServerWrapper).serveConn(0xc0196d8238, {0x8aaea08, 0xc00c2d10e0}, {0x8afd9e0, 0xc00cd73888}, {0x0, 0x4, 0x0, 0xc0201d2828, {{{{...}}, ...}, ...}})
* 	pkg/server/server_controller_sql.go:170 +0xf7
* github.com/cockroachdb/cockroach/pkg/server.(*serverController).sqlMux(0xc00697b880, {0x8aaea08, 0xc00c2d10e0}, {0x8afd9e0, 0xc00cd73888}, {0x0, 0x4, 0x0, 0xc0201d2828, {{{{...}}, ...}, ...}})
* 	pkg/server/server_controller_sql.go:90 +0x2f8
* github.com/cockroachdb/cockroach/pkg/server.startServeSQL.func1.1({0x8aaea08, 0xc010b4dc20}, {0x8afe150, 0xc011fb1e18})
* 	pkg/server/server_sql.go:1990 +0x30c
* github.com/cockroachdb/cockroach/pkg/util/netutil.(*TCPServer).ServeWith.func1({0x8aaea08, 0xc010b4dc20})
* 	pkg/util/netutil/net.go:186 +0x102
* github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2({0x71c72cc?, 0x0?})
* 	pkg/util/stop/stopper.go:498 +0x1f0
* created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 2846781
* 	pkg/util/stop/stopper.go:488 +0x47c
* 
*
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #146495 sql/tests: TestRandomSyntaxFunctions failed [test runner needs adjustment] [C-test-failure O-robot T-sql-foundations branch-master]
- #146423 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-25.2.0-rc]
- #146424 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-25.2]
- #146422 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-foundations branch-release-25.1.6-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/19685462?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/19685462?buildTab=artifacts#/) on release-25.1 @ [4841e1c252911e3e44ca4fd4942a8bbe61dfc412](https://github.com/cockroachdb/cockroach/commits/4841e1c252911e3e44ca4fd4942a8bbe61dfc412):

Fatal error:

```
panic: test timed out after 1h0m0s
running tests:
	TestRandomSyntaxFunctions (49m58s)

```
Stack: 

```
goroutine 6091762 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).processCommands(0xc006c63c08, {0x8ab1068, 0xc004627c20}, {0x0, 0x4, {0x1, {0x71a056e, 0x3}, {0xc012159570, 0xf}, ...}, ...}, ...)
* 	pkg/sql/pgwire/conn.go:252 +0x4c8
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveImpl.func4()
* 	pkg/sql/pgwire/server.go:1197 +0xec
* created by github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveImpl in goroutine 5441789
* 	pkg/sql/pgwire/server.go:1194 +0x5fb
* 
* goroutine 5679683 [chan send, 6 minutes]:
* github.com/cockroachdb/cockroach/pkg/sql.(*planner).fingerprintSpanFanout.(*planner).fingerprintSpanFanout.func1.func2({0x8aafa18, 0xc0115be640})
* 	pkg/sql/fingerprint_span.go:178 +0x3b9
* github.com/cockroachdb/cockroach/pkg/sql.(*planner).fingerprintSpanFanout.Group.GoCtx.func3()
* 	pkg/util/ctxgroup/ctxgroup.go:189 +0x8b
* golang.org/x/sync/errgroup.(*Group).Go.func1()
* 	external/org_golang_x_sync/errgroup/errgroup.go:78 +0x56
* created by golang.org/x/sync/errgroup.(*Group).Go in goroutine 5679657
* 	external/org_golang_x_sync/errgroup/errgroup.go:75 +0x96
* 
* goroutine 5755367 [chan send, 6 minutes]:
* github.com/cockroachdb/cockroach/pkg/sql.(*planner).fingerprintSpanFanout.(*planner).fingerprintSpanFanout.func1.func2({0x8aafa18, 0xc018f04f00})
* 	pkg/sql/fingerprint_span.go:178 +0x3b9
* github.com/cockroachdb/cockroach/pkg/sql.(*planner).fingerprintSpanFanout.Group.GoCtx.func3()
* 	pkg/util/ctxgroup/ctxgroup.go:189 +0x8b
* golang.org/x/sync/errgroup.(*Group).Go.func1()
* 	external/org_golang_x_sync/errgroup/errgroup.go:78 +0x56
* created by golang.org/x/sync/errgroup.(*Group).Go in goroutine 5755006
* 	external/org_golang_x_sync/errgroup/errgroup.go:75 +0x96
* 
* goroutine 5846408 [semacquire, 2 minutes]:
* sync.runtime_Semacquire(0xcb604a0?)
* 	GOROOT/src/runtime/sema.go:62 +0x25
* sync.(*WaitGroup).Wait(0x8a51c20?)
* 	GOROOT/src/sync/waitgroup.go:116 +0x48
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).serveImpl(0xc020d75d40, {0x8aafa18, 0xc0149745f0}, 0xc006e4c408, 0xc011feb728, {0x0, 0x4, {0x1, {0x71a056e, 0x3}, ...}, ...}, ...)
* 	pkg/sql/pgwire/server.go:1452 +0xa7d
* github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn(0xc020d75d40, {0x8aafa18, 0xc0149745f0}, {0x8b00040, 0xc00dfd3508}, {0x0, 0x4, 0x0, 0xc011feb6e0, {{{{...}}, ...}, ...}})
* 	pkg/sql/pgwire/server.go:958 +0xb36
* github.com/cockroachdb/cockroach/pkg/server.(*systemServerWrapper).serveConn(0xc0077723d8, {0x8ab1068, 0xc01f1eabe0}, {0x8b00040, 0xc00dfd3508}, {0x0, 0x4, 0x0, 0xc011feb6e0, {{{{...}}, ...}, ...}})
* 	pkg/server/server_controller_sql.go:170 +0xf7
* github.com/cockroachdb/cockroach/pkg/server.(*serverController).sqlMux(0xc01a097b20, {0x8ab1068, 0xc01f1eabe0}, {0x8b00040, 0xc00dfd3508}, {0x0, 0x4, 0x0, 0xc011feb6e0, {{{{...}}, ...}, ...}})
* 	pkg/server/server_controller_sql.go:90 +0x2f8
* github.com/cockroachdb/cockroach/pkg/server.startServeSQL.func1.1({0x8ab1068, 0xc01332b720}, {0x8b007b0, 0xc013f69878})
* 	pkg/server/server_sql.go:1990 +0x30c
* github.com/cockroachdb/cockroach/pkg/util/netutil.(*TCPServer).ServeWith.func1({0x8ab1068, 0xc01332b720})
* 	pkg/util/netutil/net.go:186 +0x102
* github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2({0x71c8fec?, 0x0?})
* 	pkg/util/stop/stopper.go:498 +0x1f0
* created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 3707476
* 	pkg/util/stop/stopper.go:488 +0x47c
* 
*
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #146495 sql/tests: TestRandomSyntaxFunctions failed [test runner needs adjustment] [C-test-failure O-robot P-2 T-sql-foundations branch-master]
- #146423 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot P-3 T-sql-foundations branch-release-25.2.0-rc]
- #146424 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-25.2]
- #146422 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot P-3 T-sql-foundations branch-release-25.1.6-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/tests/rsg_test.go
+++ b/pkg/sql/tests/rsg_test.go
@@ -414,7 +414,8 @@ func TestRandomSyntaxFunctions(t *testing.T) {
 					"crdb_internal.request_statement_bundle",
 					"crdb_internal.reset_activity_tables",
 					"crdb_internal.revalidate_unique_constraints_in_all_tables",
-					"crdb_internal.validate_ttl_scheduled_jobs":
+					"crdb_internal.validate_ttl_scheduled_jobs",
+					"crdb_internal.fingerprint":
 					// Skipped due to long execution time.
 					continue
 				}

